content/en/opentelemetry/compatibility.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Datadog and OpenTelemetry Compatibility
+disable_sidebar: false
+further_reading:
+- link: /opentelemetry/troubleshooting/
+  tag: "Documentation"
+  text: "OpenTelemetry Troubleshooting"
+---
+
+## Overview
+
+Datadog offers multiple setup options to accommodate various use cases, from full OpenTelemetry (OTel) implementations to hybrid setups using both OpenTelemetry and Datadog components. This page covers the compatibility between different setups and supported Datadog products and features, helping you choose the best configuration for your needs.
+
+## Setups
+
+The following setups are supported:
+
+| Setup Type                                               | API         | SDK                     | Collector/Agent                         |
+|----------------------------------------------------------|-------------|-------------------------|-----------------------------------------|
+| **[Full OpenTelemetry][7]**                              | OTel API    | OTel SDK                | OTel Collector                          |
+| **[OTel to Datadog Agent (OTLP)][6]**                    | OTel API    | OTel SDK                | Datadog Agent (OTLP Ingest)             |
+| **[OTel API with Datadog SDK and Agent][8]**             | OTel API    | Datadog SDK             | Datadog Agent                           |
+| **Datadog Agent with embedded OTel Collector (Preview)** | OTel API    | OTel SDK or Datadog SDK | Datadog Agent (with embedded Collector) |
```

Comment:
```suggestion
| **Datadog Agent with embedded OTel Collector (Preview)** | OTel API    | OTel SDK or Datadog SDK | Datadog Agent (with embedded Collector) |
```
Not a huge deal but just making sure these last 2 items don't have something they should link to ?